deployment/postgres-init-db/init-aerie.sh

```diff
@@ -17,6 +17,11 @@ psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-E
   GRANT ALL PRIVILEGES ON DATABASE aerie_merlin TO aerie;
   \echo 'Done!'
 
+  \echo 'Initializing aerie_sched database...'
+  CREATE DATABASE aerie_sched;
```

Comment:
:large_blue_circle: Is there are reason we dropped the "uler" off aerie_sched(uler) here? What happened Paul? I thought you and I both loved more verbose naming over shortening. 😆

Reply:
lol normally I'd use `aerie_scheduler` here but this was just to be consistent with the `docker-compose.yaml`'s name for the DB (committed by Steve some time ago I think):
https://github.com/NASA-AMMOS/aerie/blob/181e751451dbc2fd0f1068d54d745d7c68e4f8e4/deployment/docker-compose.yml#L45

Reply:
Ahh true. That's fair, I prefer the longer version, but yes I follow your reasoning. If you want to change it I'd support that 😃, but no pressure.

Reply:
Actually, I'll tag this on as a commit to this PR. Would be nice to keep this commit separate from the first 2 commits though, that way the name change and the root of the PR are semantically separated.

Reply:
🥳. Agreed.

Reply:
Done :) Also renamed some env. vars.